index.html

```diff
@@ -108,63 +128,74 @@ <h4>Profesional Art Printing Data</h4>
         <li>Ruby</li>
       </ul>
         <button class="see-button" type="button">See project</button>
-    </section>
-    </section>
+  </section>
+  </div>
     <section class="aboutme">
-      <div>
+      <div class="aboutme-header">
+        <div class="aboutme-info">
         <h2>About me</h2>
-      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
       <button class="get_my-resume" type="button">Get My Resume</button>
-      </div>
+    </div>
       <div class="side-image"></div>
+    </div>
       <hr class="aboutme-line">
       <div class="techno">
-        <div class="all">
-          <img src="images/Diagonal.png" alt="square-image">
-          <h5>Languages</h5>
+        <div class="aboutme-Languages">
+          <div class="all">
+            <img src="images/Diagonal.png" alt="square-image">
+            <h5>Languages</h5>
+          </div>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
```

Comment:
- [ ] Congratulation, you have done an extreme job on this website, but I still can think that there is some issue that you need to handle, adjusting your CSS properties to align these item correctly, see image below 
- 
<img width="1019" alt="hav" src="https://user-images.githubusercontent.com/79058364/154361347-5d8848e1-c0bd-44de-98a0-e859fdc24a22.png">